### PR TITLE
Added a resources file where the community can link to external workshops, blog posts, etc

### PR DESCRIPTION
**Pull request by sarahmaddox**
This PR creates a place in our docs where the community can link to bog posts, workshops, codelabs, videos, and other resources that are hosted outside the Kubeflow repositories and docs. The PR also sets up the Hugo content blocks and some styling for the new resources page.

After this PR is merged, it'd be great if anyone from the community has any ideas about further improving the styling!

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/449)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
/hold

Before merging this PR, I'd like some community input on the page in general. For example:

* Do you like the idea of such a page? 
* Do you have items that you'd like to add to it immediately? I'll do that if you add a link to the item plus a short description in the comments on this PR. Alternatively, you can create a PR to add more items later.
* Are there any additional content types we should add? We have videos, blog posts, codelabs, etc...

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-449--competent-brattain-de2d6d.netlify.com/docs/examples/resources/

**Comment by gaocegege** [comment]
I love this idea. Actually, there are also some blog posts in Chinese, could we add them in this page?

/lgtm

**Comment by sarahmaddox** [comment]
@gaocegege Thanks! Yes, we can add blog posts that are in Chinese. After this PR is merged, please would you send me a PR to add the links, with a short description of each blog post in English.

**Comment by gaocegege** [comment]
Thanks, LGTM

File: content/docs/examples/resources.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+ (using Jupyter notebooks).
+{{% /blocks/content-item %}}
+{{< /blocks/content-section >}}
+
+
+{{< blocks/content-section title="Videos" color="white" >}}
+
+{{% blocks/content-item %}}
+Tutorials and overviews published in video format. Be aware that a video 
+describes the interfaces at the time of publication of the video.
```
Comment (dwightjl) [inline content/docs/examples/resources.md]: Perhaps also clarify that interfaces change frequently?

File: content/docs/examples/resources.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+This example demonstrates how you can use Kubeflow to train and serve a 
+distributed Machine Learning model with PyTorch on a Google Kubernetes Engine 
+cluster in Google Cloud Platform (GCP).
+{{% /blocks/content-item %}}
+
+
+{{% blocks/content-item title="Getting started with Kubeflow Pipelines"
+  date="November 16, 2018"
+  url="https://cloud.google.com/blog/products/ai-machine-learning/getting-started-kubeflow-pipelines" %}}
+In this article, we’ll describe how you can tackle ML workflow operations with
```
Comment (dwightjl) [inline content/docs/examples/resources.md]:
```suggestion
This article describes how you can tackle ML workflow operations with 
```

File: content/docs/examples/resources.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+distributed Machine Learning model with PyTorch on a Google Kubernetes Engine 
+cluster in Google Cloud Platform (GCP).
+{{% /blocks/content-item %}}
+
+
+{{% blocks/content-item title="Getting started with Kubeflow Pipelines"
+  date="November 16, 2018"
+  url="https://cloud.google.com/blog/products/ai-machine-learning/getting-started-kubeflow-pipelines" %}}
+In this article, we’ll describe how you can tackle ML workflow operations with 
+Kubeflow Pipelines, and then we’ll highlight some examples that you can try
```
Comment (dwightjl) [inline content/docs/examples/resources.md]:
```suggestion
Kubeflow Pipelines, and highlights some examples that you can try 
```

File: content/docs/examples/resources.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+{{% blocks/content-item %}}
+This section introduces the examples in the 
+[kubeflow/examples](https://github.com/kubeflow/examples) repo.
+{{% /blocks/content-item %}}
+
+{{% blocks/content-item title="Semantic code search"
+  url="https://github.com/kubeflow/examples/tree/master/code_search" %}}
+Perform a semantic code search using a Sequence to Sequence natural language 
+processing model. This tutorial runs in a Jupyter notebook and uses Google
```
Comment (dwightjl) [inline content/docs/examples/resources.md]:
```suggestion
perform a semantic code search. This tutorial runs in a Jupyter notebook and uses Google 
```

Comment (dwightjl) [reply content/docs/examples/resources.md]: Line 2 of the previous suggested edit.

File: content/docs/examples/resources.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+{{< blocks/content-section title="Kubeflow samples" color="light" >}}
+
+{{% blocks/content-item %}}
+This section introduces the examples in the 
+[kubeflow/examples](https://github.com/kubeflow/examples) repo.
+{{% /blocks/content-item %}}
+
+{{% blocks/content-item title="Semantic code search"
+  url="https://github.com/kubeflow/examples/tree/master/code_search" %}}
+Perform a semantic code search using a Sequence to Sequence natural language
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/examples/resources.md]: Thanks. FYI, the "suggested change" feature doesn't work well on a Mac - I can only see 1 line in the little window, and no way of expanding/scrolling. I'm having to copy the text blind, then figure out what the change should be.

File: content/docs/examples/resources.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+{{% /blocks/content-item %}}
+
+{{% blocks/content-item title="OpenShift Kubeflow Workshop"
+  url="https://github.com/AICoE/openshift_kubeflow_workshop" %}}
+Run Kubeflow on Red Hat [OpenShift](https://www.openshift.com/).
+{{% /blocks/content-item %}}
+
+{{% blocks/content-item title="Katacode scenarios" 
+  url="https://www.katacoda.com/kubeflow" %}}
+Follow these tutorials on deploying Kubeflow and running a machine learning
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/examples/resources.md]: Done

File: content/docs/examples/resources.md (hunk 2 above)
Comment (sarahmaddox) [reply content/docs/examples/resources.md]: Done

**Comment by sarahmaddox** [comment]
Thanks for the review @dwightjl!

**Comment by abhi-g** [comment]
Excellent idea!

/approve

**Pull request merged**